packages/flutter_tools/test/general.shard/devtools_launcher_test.dart

```diff
@@ -153,3 +237,7 @@ void main() {
     expect(logger.errorText, contains('Failed to launch DevTools: ProcessException'));
   });
 }
+
+
+class MockPlatform extends Mock implements Platform {}
```

Comment:
Use a `FakePlatform` instead of a `Mock`

Reply:
done